src/components/WebexAvatar/WebexAvatar.js

```diff
@@ -1,6 +1,41 @@
-import React from 'react';
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
 import {Avatar} from '@momentum-ui/react';
 
-export default function WebexAvatar(props) {
-  return <Avatar {...props} />;
+const propTypes = {
+  personID: PropTypes.string.isRequired,
+  adapter: PropTypes.object.isRequired,
+};
+
+export default class WebexAvatar extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.props = props;
+    this.state = {
+      person: {},
+    };
+  }
+
+  componentDidMount() {
+    this.props.adapter.getPerson(this.props.personID).subscribe(
+      (person) => {
+        this.setState({person});
+      },
+      () => {
+        this.setState({
+          person: {
+            displayName: 'X X',
```

Comment:
Can we come up with an agreed state that represents the avatar being in an error state? I didn't see one mentioned in the ticket @lalli-flores

Reply:
I removed it